### PR TITLE
Add coverage reporting to code climate

### PR DESCRIPTION
**Pull request by eckdanny-osi**
Closes [CASE-1088](https://osi-cwds.atlassian.net/browse/CASE-1088)

### CODE REVIEW

**Comment by SashaKyz** [comment]
Can one of the admins verify this patch?

**Comment by eckdanny-osi** [comment]
No metrics showing up in code-climate yet, but i think in order to show things like coverage up/down, coverage threshold violations, etc. there needs to be base rate data from `master`.

I'd remove the **WiP**, but looks like i forget to ever add the prefix 😒

File: JenkinsfileCodeReview [hunk 1]
```diff
@@ -1,12 +1,18 @@
 DOCKER_GROUP = 'cwds'
 DOCKER_IMAGE = 'casemanagement'
+CC_TEST_REPORTER_ID = 'e90a72f974bf96ece9ade12a041c8559fef59fd7413cfb08f1db5adc04337197'
```
[the remark this answers is not in the document]
Comment (edgibbs) [reply JenkinsfileCodeReview]: Oh and we should probably regenerate at that point

**Comment by edgibbs** [comment]
Merging to test, will swap out CC token for a new one with a environment variable if it works.

**Pull request merged**